android/src/main/java/com/reactnativefacetec/ZoomProcessors/ZoomGlobalState.java

```diff
@@ -24,6 +26,23 @@
         "8QIDAQAB\n" +
         "-----END PUBLIC KEY-----";
 
+    // OUR (CHIPPER) KEY
+    // public static String PublicFaceMapEncryptionKey = 
+    //     "-----BEGIN PUBLIC KEY-----\n" +
+    //     "MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAv+HDfn7oE26J3g9LRS4D\n" +
+    //     "FtFiXMigTUarU1GZi8Q487KHxb4mxlPIroY0nOerMZRZr6T6ltKwRO6n6Izld4Ji\n" +
+    //     "ukjaSF6UcWI+i7NXHqyumO8VcReJscSoV/aGaQNBbenh6seX5+XmOE5erlAw1QqQ\n" +
+    //     "5Nmji/MSMrNXbWKgSWYYkwzDyth9Yi6KQX2vuhebcduPGnqr3Mi9YXOCtUgAJ83A\n" +
+    //     "qx0dcmwZ74ysfaUsgoKbRDQxT96Ff0nto6CfXExRG4J9EtQLJS0E4E6ztl0ElwOH\n" +
+    //     "iNTG3LF46T8PH64rBllt/ejDVZlQIOeRUjNBjnfJ3+0IkBlLLCMmTtvgCJ5kPg3k\n" +
+    //     "twIDAQAB\n" +
+    //     "-----END PUBLIC KEY-----";
+
+    public static String LicenseText = 
+        "appId      = com.chippercash\n" +
+        "expiryDate = 2020-11-10\n" +
+        "key        = 00304502200297458d060622f0be36dc78a86a3428370081250b6bfcf5e1013477f0562608022100e84bfb972bab94042ddd840bf6beafc187cb37cc7eb2473739cafee661ef6684";
```

Comment:
Before we go live - this key would need to come from Compliance, we won't want to hardcode this because we wouldn't be able to update it.